ParisBNF/abb/BNFabb51.xml

```diff
@@ -0,0 +1,495 @@
+                                </extent>
+
+                                <foliation>Foliation in Eastern Arabic numerals
+                                  in the upper right corner of the folia.
+                                </foliation>
+
+                                <collation>
+                                   <list>
+                                      <item xml:id="q1" n="1">
+                                         <dim unit="leaf">8</dim>
+                                         <locus from="1r" to="8v"/>
+                                       </item>
+
+                                      <item xml:id="q2" n="2">
+                                        <dim unit="leaf">8</dim>
+                                        <locus from="9r" to="16v"/>
+                                        </item>
+
+                                        <item xml:id="q3" n="3">
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="17r" to="24v"/>
+                                          </item>
+
+                                        <item xml:id="q4" n="4">
+                                        <dim unit="leaf">8</dim>
+                                      <locus from="25r" to="32v"/>
+                                        </item>
+
+                                        <item xml:id="q5" n="5">
+                                        <dim unit="leaf">8</dim>
+                                      <locus from="33r" to="40v"/>
+                                        </item>
+
+                                        <item xml:id="q6" n="6">
+                                        <dim unit="leaf">6</dim>
+                                      <locus from="41r" to="46v"/>
+                                        </item>
+
+                                        <item xml:id="q7" n="7">
+                                        <dim unit="leaf">8</dim>
+                                      <locus from="47r" to="54v"/>
+                                        </item>
+
+                                        <item xml:id="q8" n="8">
+                                        <dim unit="leaf">6</dim>
+                                      <locus from="55r" to="60v"/>
+                                        </item>
+
+                                        <item xml:id="q9" n="9">
+                                        <dim unit="leaf">6</dim>
+                                      <locus from="61r" to="66v"/>
+                                        </item>
+
+                                        <item xml:id="q10" n="10">
+                                        <dim unit="leaf">10</dim>
+                                      <locus from="67r" to="74v"/>
+                                        </item>
+                                        <item xml:id="q11" n="11">
+                                        <dim unit="leaf">3</dim>
```

Comment:
any information on stubs? or diagrams won't be possible

Reply:
done